### PR TITLE
Doc 6.0 update

### PR DESCRIPTION
**Pull request by arthurhjorth**

### CODE REVIEW

**Comment by arthurhjorth** [comment]
@qiemem how's this looking? Is this sufficient? SHould we add any other general usecases?

File: USING.md [hunk 1]
```diff
@@ -2,23 +2,41 @@
...
 
-LevelSpace has two different child model types, headless models and interactive models. They each have their strengths and weaknesses: Headless models are faster and use less memory than interactive models. Interactive models allow you full access to their interface and widgets, whereas headless models only give you access to their view and command center. Typically you will want to use headless models when you are running a large number of models, or if you simply want to run them faster - interactive models are good if you run a small amount of models, or if you are writing a LevelSpace model and need to be able to debug.
+In general, the LevelSpace syntax has been designed to align with existing NetLogo primitives whenever possible.
+
+### Headless and Interactive Models
```
Comment (qiemem) [inline USING.md]: Should we say that interactive are visible by default and headless are hidden?

File: USING.md [hunk 2]
```diff
@@ -2,23 +2,41 @@
...
+Typically you will want to use headless models when you are running a large number of models, or if you simply want to run them faster. Interactive models are good if you run a small amount of models, if you are writing a LevelSpace model and need to be able to debug, or if you need access to widgets during runtime.
+
+### Keeping Track of Models
 
 Child models are kept track of in the extension with an id number, starting with 0, and all communication from parent to child is done by referencing this number, henceforth referred to as `model-id`.
 
 The easiest way to work with multiple models is to store their `model-id` in a list, and use NetLogo's list primitives to sort, filter, etc. them during runtime.
 
-### A general usecase
+Keeping track of models is important: Most LevelSpace primitives will fail and cause a runtime interruption if provided a `model-id` to a non-existing model.
```
Comment (qiemem) [inline USING.md]: I don't think this is necessary. Does it ever really come up?

File: documentation.conf [hunk 3]
```diff
@@ -54,13 +54,13 @@ primitives = [
...
 
-If given a command, LevelSpace will call the command after loading each instance of the model withe model id as the argument. This allows you to easily store model ids in a variable or list when loading models, or do other initialization. For example, to store a model id in a variable, you can do:
+If given a command, LevelSpace will call the command after loading each instance of the model with the `model-id` as the argument. This allows you to easily store model ids in a variable or list when loading models, or do other initialization. For example, to store a model id in a variable, you can do:
 
 ```NetLogo
 let model-id 0
-(ls:create-models "My-Model.nlogo" [ set model-id ? ])
+(ls:create-models "My-Model.nlogo" [ [id] -> set model-id id ])
```
Comment (qiemem) [inline documentation.conf]: Damn, how did I miss this?

File: USING.md (hunk 1 above)
Comment (arthurhjorth) [inline USING.md]: Yeah, we say it in documentation.conf but it might be good here too. I'll do taht.

File: USING.md (hunk 2 above)
Comment (arthurhjorth) [inline USING.md]: Absolutely, especially for novices. In fact, now that I think about it, we should add a usecase for updating lists to keep them up to date :p

**Comment by qiemem** [comment]
Looks like you accidentally checked your vim swap file: `.documentation.conf.swp`, so get rid of that.

You can regenerate `README.md` with `./sbt extensionDocument`.

**Comment by qiemem** [comment]
Looks great overall!

File: USING.md (hunk 2 above)
Comment (qiemem) [inline USING.md]: Oh, I completely forgot about manually created "modelsets"... hmm that's actually a really nasty problem.

**Comment by arthurhjorth** [comment]
I think I built the documentation. But I had to add quotation marks to the `model-exists?` prim because it contains a question mark. Can you please check if it worked out okay? I think everything should be good now.

**Pull request merged**